src/main.ts

```diff
@@ -3,7 +3,7 @@ import Stoa from './Stoa';
 import express from "express";
 
 const port: number = Number(process.env.PORT) || 3836;
-const stoa: express.Application = new Stoa().stoa;
+const stoa: express.Application = new Stoa(".cache").stoa;
```

Comment:
```suggestion
const stoa: express.Application = new Stoa("database").stoa;
```

Don't use `.cache`, that's what Agora do, and at the very least don't do the same thing as Agora to avoid conflict if someone inadvertently runs agora AND stoa in the same working directory. It shouldn't be the case, but if it happens, let's remove the obvious pitfalls.

Reply:
I apply the suggestion.